core/src/main/java/nl/nn/adapterframework/filesystem/FileSystemSender.java

```diff
@@ -104,18 +104,15 @@ public PipeRunResult sendMessage(Message message, PipeLineSession session, IForw
 
 		try {
 			if (paramList !=null) {
-				pvl=paramList.getValues(message, session);
+				pvl = paramList.getValues(message, session);
 			}
 		} catch (ParameterException e) {
 			throw new SenderException(
 					getLogPrefix() + "Sender [" + getName() + "] caught exception evaluating parameters", e);
 		}
 
 		try {
-			Object result=actor.doAction(message, pvl, session);
-			if (result instanceof PipeRunResult) {
-				return (PipeRunResult)result;
-			}
```

Comment:
Nu begin ik het wel eng te vinden, waar komt dit vandaan...

Reply:
Uhm, ja, dat zat er in...  Volgens mij kon het nooit (meer?) een PipeRunResult zijn. Misschien ooit, vroeger, wel?

Reply:
`Message` kan niet worden gecast naar `PipeRunResult` dus nu zou het een syntax-error zijn.
Ik heb nog even gecontroleerd maar ik zie nergens waar de code een `PipeRunResult` zou kunnen teruggeven, ook niet als return van een andere method, wat nu eventueel zou worden gewrapped als `Message` ... Ik zag strings en streams. De tests gingen ook nergens uit van een `PipeRunResult`, maar alleen van strings of dingen die naar Message konden worden omgezet.